lib/smart_answer_flows/marriage-abroad/outcomes/os_affirmation/_fees.govspeak.erb

```diff
@@ -4,7 +4,10 @@
 
 <% unless calculator.country_without_consular_facilities? ||
    calculator.ceremony_country == 'indonesia' ||
-   calculator.ceremony_country == 'monaco' %>
+   calculator.ceremony_country == 'monaco' ||
+   calculator.ceremony_country == 'france' ||
+   calculator.ceremony_country == 'new-caledonia' ||
+   calculator.ceremony_country == 'wallis-and-futuna' %>
```

Comment:
As you've already spotted, this happens in a later commit titled "DRY up condition for link to consular fees".